Core/SaveState.cpp

```diff
@@ -621,7 +604,14 @@ namespace SaveState
 
 			case SAVESTATE_SAVE:
 				INFO_LOG(SAVESTATE, "Saving state to %s", op.filename.c_str());
-				result = CChunkFileReader::Save(op.filename, g_paramSFO.GetValueString("TITLE"), PPSSPP_GIT_VERSION, state);
+				title = g_paramSFO.GetValueString("TITLE");
+				if (title.empty()) {
+					// Homebrew title
+					title = PSP_CoreParameter().fileToStart;
+					std::size_t lslash = title.find_last_of("/");
```

Comment:
I'm checking here for the last "/" to get what's after it - folder name or filename(in case of standalone elf that's not a bootable folder). :) See one line above and under.
 Althrough that part as a whole is just cosmetic - title for homebrew ss which can be seen in ss manager.